### ISSUE TITLE
Is card.io still active?

### ISSUE THREAD

**Issue by vpontis** [Kerumen/react-native-awesome-card-io#150]
Their website is down and it looks like they archived all of their Github repos...

**Comment by Kerumen** [comment]
Yes unfortunately. I tried to contact them but got no answer.. As this library is only a wrapper around the official SDKs, nothing much I can't do on my side :(

**Comment by SalahAdDin** [comment]
Is it open to user?
